src/component/utils/camelCaseConverter.tsx

```diff
@@ -0,0 +1,7 @@
+export function camelCaseToNormalText(input: string): string {
```

Comment:
why do we need this function?

Reply:
We have media type named as busShelter to convert it into Bus Shelter and show in the card and also in Filter we can use this to show the values properly in UI because from the backend it will be sent as below
"lighting": [
        "backLit",
        "nonLit",
        "frontLit",
        "led"
    ]

Reply:
that is not the right way to do it.
create a map manually here.
```suggestion
{
nonLit: "Non Lit",
busShelter: "Bus Shelter"
}
```